src/test/java/com/wikia/webdriver/testcases/desktop/discussions/DeleteAllPostsByUserTests.java

```diff
@@ -40,15 +39,15 @@
    * @param wikiName wiki on which a post by `userWithPosts` will be created
    * @return post that was created
    */
-  private PostEntity.Data setUp(String wikiName) {
-    siteId = Utils.excractSiteIdFromWikiName(wikiName);
+  private PostEntity.Data setUp(String wikiName, String language) {
+    siteId = Utils.extractSiteIdFromWikiName(wikiName, language);
     return DiscussionsClient
       .using(userWithPosts, driver)
       .createPostWithUniqueData(siteId);
   }
 
   private PostEntity.Data setUp() {
-    return setUp(MobileWikis.DISCUSSIONS_4);
+    return setUp("qadiscussions", "de");
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Define a constant instead of duplicating this literal "qadiscussions" 3 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AS1192)